x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/SearchableSnapshotActionTests.java

```diff
@@ -47,15 +48,29 @@ public void testToSteps() {
         assertThat(steps.get(12).getKey(), is(expectedSteps.get(12)));
         assertThat(steps.get(13).getKey(), is(expectedSteps.get(13)));
         assertThat(steps.get(14).getKey(), is(expectedSteps.get(14)));
+        assertThat(steps.get(15).getKey(), is(expectedSteps.get(15)));
```

Comment:
nit: what's the deal with all this unrolling?

Reply:
I mainly followed the same methodology. I will leave it to @dakrone to comment on the idea behind this?

Reply:
There's no reason these have to be unrolled, as long as we have the expected steps comparing them all at once is fine. I don't know why they were initially unrolled.